### ISSUE TITLE
(gen_img[:, 0, 0, 0] * 0 + loss_Gpl) ... but why?

### ISSUE THREAD

**Issue by Mwni** [NVlabs/stylegan2-ada#103]
In `loss.py`, why is this being done?

```
(gen_img[:, 0, 0, 0] * 0 + loss_Gpl).mean().mul(gain).backward()
```
and
```
(real_logits * 0 + loss_Dreal + loss_Dr1).mean().mul(gain).backward()
```
both times, a term multiplied by zero.

What did they mean by this?

**Comment by Mwni** [comment]
I am in the wrong repo.
